src/Trixi.jl

```diff
@@ -140,6 +140,7 @@ export AcousticPerturbationEquations2D,
        CompressibleEulerMulticomponentEquations1D,
        CompressibleEulerMulticomponentEquations2D,
        IdealGlmMhdEquations1D, IdealGlmMhdEquations2D, IdealGlmMhdEquations3D,
+       ViscoResistiveMhd3D,
```

Comment:
```suggestion
       ViscoResistiveMhdEquations3D,
```

(for consistency with the other names)